### ISSUE TITLE
Join forces with `kirby-fest` package?

### ISSUE THREAD

**Issue by johannschopplich** [benwest/kql-ts#1]
Hey there,

that's some stunning TS magic you created. 🪄 I have created a repo for basic Kirby types – [kirby-fest](https://github.com/johannschopplich/kirby-fest). 
Inferring the query result based on the actual query is incredible. I'd love to use that for my [Nuxt KQL](https://nuxt-kql.jhnn.dev/) module and across all of my projects…

Wanna join forces? 🙂

**Comment by benwest** [comment]
Yes of course! I’ll be tinkering with this over the next month or two in small projects. I still expect to find some mistakes, and as I mention in the repo, the `models.ts` file is very much not complete. Anyway you’re welcome to use it for whatever. I guess I should put a license here?

Fire away if you have any ideas. I wonder if you can try it out in some more complex sites?
